### PR TITLE
Assign fallback whatsnew page content to non-FRENDE locales for Firefox 70 (Fixes #8039)

### LINKED ISSUE #8039: WNP 70.0 - non-FRENDE locales - assign old WNP fallback content w. Mobile CTA (widget or QR code)
## Description

The ETP in the non-FRENDE locales for WNP70 is not working to advance the building of relationships/cross-promote other products and services.

The ask here is to leverage the [existing fallback content in the WNP with the mobile CTA](https://www.mozilla.org/it/firefox/66.0/whatsnew/all/), widget and QR code, and assign that to the WNP 70 URL for all locales EXCEPT fr, de, en-US, en-CA & en-GB

Instrumentation in the page will need to be updated such that params will reflect the page/version.

NOTE: The hardcoded Indonesian WNP template promoting Lite still needs to be shown as it is.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Non-FRENDE locales are shown the fallback WNP content when updating from a <70.0 version of Firefox to the 70.0 release.

#### Tasks
- [ ] Assign the fallback content for WNP to all locales EXCEPT fr, de, en-US, en-CA & en-GB to be shown at https://www.mozilla.org/firefox/70.0/whatsnew/all/ 
- [ ] Update page instrumentation

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
Non-FRENDE locales are shown the fallback WNP content when updating from a <70.0 version of Firefox to the 70.0 release.

## Issue / Bugzilla link
#8039

## Testing
http://localhost:8000/firefox/70.0/whatsnew/all/
- [x] Locales (except DE, FR, EN-*) receive fallback whatsnew page template featuring mobile CTA.
- [x] Removes the non-locale-specific /whatsnew 70 template and related assets.

### CODE REVIEW

File: bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html [hunk 1]
```diff
@@ -1,115 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (alexgibson) [inline bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html]: I'm not entirely clear as the issue doesn't mention this, but why is this PR removing this template?

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70.html [hunk 2]
```diff
@@ -1,110 +0,0 @@
...
-      Read the <a href="{{ notes }}">Release Notes</a> to know more about what’s new in your Firefox Browser.
-    {% endtrans %}
-    </p>
-  </aside>
-
-</main>
-{% endblock %}
-
-{% block js %}
-  {{ js_bundle('firefox_whatsnew_70') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx70.html]: Can this JS bundle be removed?

File: bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html (hunk 1 above)
Comment (amychurchwell) [reply bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html]: I think that was my own misunderstanding. Adjusting PR. :+1:

**Comment by amychurchwell** [comment]
@alexgibson 🤦‍♀ ah jeeze. whoops.

**Pull request merged**